docs/releases/1.11.1.txt

```diff
@@ -45,3 +45,5 @@ Bugfixes
 
 * Fixed exception reraising in ORM query execution when ``cursor.execute()``
   fails and the subsequent ``cursor.close()`` also fails (:ticket:`28091`).
+
+* Fixed ``ChoiceWidget`` to not localize option values (:ticket:`28075`).
```

Comment:
ChoiceWidget isn't a public API, so perhaps it would be better to say something like "Fixed a regression where A, B, C, and D localized option values" where the letters are the actual classes.